spec/helpers/http_mock.lua

```diff
@@ -43,11 +44,12 @@ local function default_field(tbl, key, default)
 end
 
 -- create a mock instance which represents a HTTP mocking server
--- @param listens: the listen directive of the mock server, defaults to "0.0.0.0:8000"
--- @param code: the code of the mock server, defaults to a simple response.
--- @param opts: options for the mock server, left it empty to use the defaults
--- @return: a mock instance
--- @usage
+---@param listens table|string|number|nil: the listen directive of the mock server, defaults to a random available port
```

Comment:
what's up with these doc-comments? they should be `ldoc` based, but this is a different format.

The old format was correct; `-- @param ...`

Reply:
Got a little bit confused as I can find both formats in our code base. The `---@param` works with my VSC plugin's hover-over hint so I chose this.
Changing to the correct format.